documentation/guides/migrating-from-v4-to-v5.md

```diff
@@ -0,0 +1,158 @@
+# Migrating from v4 to v5
+
+Polaris v5.0.0 ([full release notes](https://github.com/Shopify/polaris-react/releases/tag/v5.0.0)) removes most features deprecated in v4.x.x releases. This file describes all code updates required to stay up to date.
+
+## AppBridge integrations removed
+
+In v3 and v4 `AppProvider` could accept `apiKey` `shopOrigin` and `forceRedirect` props that would configure AppBridge delegation in our `ResourcePicker`, `Loading`, `Modal`, `Page` and `Toast` components.
+
+The `ResourcePicker` component has been fully removed is it only had meaning within an AppBridge app. Other components retain their non-app-bridge behaviour.
+
+AppBridge integration has been removed and abstracted into a separate `@shopify/app-bridge-react` package. If you require the AppBridge implementation of these components then use the components from the [`@shopify/app-bridge-react` package](https://shopify.dev/tools/app-bridge/react-components).
+
+```js
+// Old
+import {AppProvider, Modal} from '@shopify/polaris';
+
+function MyApp() {
+  const config = {apiKey: '12345', shopOrigin: shopOrigin};
+
+  return (
+    <AppProvider apiKey={config.apiKey} shopOrigin={config.shopOrigin}>
+      <Modal /* ... *//>
+    </AppProvider>
+  );
+}
+
+
+// New
+import {AppProvider} from '@shopify/polaris';
+import {Provider, TitleBar} from '@shopify/app-bridge-react';
+
+function MyApp() {
+  const config = {apiKey: '12345', shopOrigin: shopOrigin};
+
+  return (
+    <AppProvider>
+      <Provider config={config}>
+        <Modal /* ... *//>
+      </Provider>
+    <AppProvider>
+  );
+}
+```
+
+## AppProvider's i18n translations array is now preferred language first
+
+AppProvider's i18n prop accepts an array of translation dictionaries so that if
+a translation key in one language is not found it can try a fallback language.
```

Comment:
```suggestion
AppProvider's i18n prop accepts an array of translation dictionaries so that if a translation key in one language is not found it can try a fallback language.
```